core/mutator/mutator.go

```diff
@@ -55,6 +55,7 @@ type Mutator interface {
 // Mutation reads and writes mutations to the database.
 type Mutation interface {
 	// Read reads all mutations for a specific given mapID, epoch, and index.
+	// TODO(cesarghali): Add mapID to this interface.
```

Comment:
Why do we need `mapID` here? Mutation objects are initialized with a map ID. If this is related to supporting multiple maps per server, can you please add that to the comment?